libs/langchain/langchain/vectorstores/redis.py

```diff
@@ -368,6 +368,29 @@ def similarity_search_with_score(
             docs_and_scores.append((doc, float(result.vector_score)))
         return docs_and_scores
 
+    def similarity_search_with_score_with_limit_score(
```

Comment:
convention with other vector stores has been to call this ..._with_relevance_score. ik this already has some _with_limit_score methods but not sure we want to keep extending them

Reply:
Good point. Yeah, vector stores are getting complex with each classes implementing separate methods to filter by score threshold.
How about fixing this Redis  to match the base `similarity_search_with_relevance_scores` that takes score_threshold from `**kwargs`?

https://github.com/langchain-ai/langchain/blob/82df923f3737c053799db2dca99aa85921a15059/libs/langchain/langchain/vectorstores/base.py#L228:L273

Reply:
Fixed
https://github.com/langchain-ai/langchain/pull/8116/commits/ccc608aaea6da344cee4e0685c3301650fab27a0